client/components/product-expiration/README.md

```diff
@@ -13,6 +13,9 @@ This controls the formatting of the date format, which is passed to a localized
 #### `expiryDateMoment { moment }`
 This is the expiration date as a `moment` instance, and is required for display.
 
+#### `renewDateMoment { moment }`
+This is the date on which the product will be renewed, as a `moment` instance, used only when provided.
```

Comment:
(minor) It might be more clear to say "will be auto-renewed" here.

Reply:
Thanks. Implemented.